shared/src/main/scala/mouse/foption.scala

```diff
@@ -87,4 +88,7 @@ final class FOptionOps[F[_], A](private val foa: F[Option[A]]) extends AnyVal {
 
   def liftOptionT: OptionT[F, A] =
     OptionT(foa)
+
+  def liftEitherT[E](ifNone: => E)(implicit F: Functor[F]): EitherT[F, E, A] =
```

Comment:
Well, if we have the `liftOptionT` in the `foption` syntax already, then probably `liftEitherT` is fine too.